infra/op-replica/docker-compose/default-mainnet.env

```diff
@@ -14,6 +13,8 @@ L2GETH_HTTP_PORT=9991
 L2GETH_WS_PORT=9992
 DTL_PORT=7878
 GETH_INIT_SCRIPT=check-for-chaindata-berlin.sh
+HEALTHCHECK__REFERENCE_RPC_PROVIDER=https://mainnet.optimism.io
```

Comment:
We might want to ask people to bring their own L2 RPC endpoints rather than pointing them at mainnet.optimism.io. More and more people are running nodes, so this will start to get taxing on the public endpoint.